### ISSUE TITLE
Column location request for "Add a card" modal pop-up  

### ISSUE THREAD

**Issue by jtmusky** [imdone/imdone#49]
Navigating with keyboard only, if a column is empty, unable to navigate to the column. Using the keyboard shortcut "space" to add a new card is only applicable to non-empty columns

**Describe the solution you'd like**
Suggest adding another keyboard shortcut "i.e. (Shift + space)" to pop-up a dialog of which column to add the new card. Possibly use numbers to designate columns for a one key action,

**Describe alternatives you've considered**
Maybe use a number to preposition which column to use before pressing "space". Something like the following:
`4, space` This command will popup the new card dialog for the 4th column.
`12, space` this will popup the new card dialog for the 12th column.

**Additional context**

**Comment by piascikj** [comment]
implemented in 1.11.0
